File: app/src/main/java/ua/meugen/android/pdfcomposer/model/db/Converters.java
package ua.meugen.android.pdfcomposer.model.db;

import android.arch.persistence.room.TypeConverter;

import java.util.Date;

public class Converters {

    @TypeConverter
    public static long dateToMillis(final Date date) {
        return date.getTime();
    }

    @TypeConverter
    public static Date millisToDate(final long millis) {
        return new Date(millis);
    }
}

File: app/src/main/java/ua/meugen/android/pdfcomposer/model/db/Converters.kt
package ua.meugen.android.pdfcomposer.model.db

import android.arch.persistence.room.TypeConverter

import java.util.Date

object Converters {

    @JvmStatic
    @TypeConverter
    fun dateToMillis(date: Date): Long {
        return date.time
    }

    @JvmStatic
    @TypeConverter
    fun millisToDate(millis: Long): Date {
        return Date(millis)
    }
}
